index.jsx

```diff
@@ -1 +1,38 @@
-export default () => {};
+const toString = (element) => {
+  if (typeof element === 'string') {
+    return element;
+  }
+
+  const name = typeof element.type === 'string'
+    ? element.type
+    : element.type.name;
+
+  if (!name) {
+    throw new Error('Component name must be set');
+  }
+
+  const attributes = Object.keys(element.attributes)
+    .map((key) => {
+      const type = typeof element.attributes[key];
+      let value;
+
+      if (type.toLowerCase() === 'string') {
+        value = `'${element.attributes[key]}'`;
+      } else if (type.toLowerCase() === 'boolean' || type.toLowerCase() === 'number') {
```

Comment:
`typeof` is always lower case already so these `.toLowerCase()` methods shouldn't be needed.